index.js

```diff
@@ -38,13 +38,21 @@ module.exports = (opts) => {
         });
 
         comp.hooks.done.tap('WebpackServe', (stats) => {
-          const json = stats.toJson();
-          if (stats.hasErrors()) {
-            bus.emit('compiler-error', { json, compiler: comp });
-          }
-
-          if (stats.hasWarnings()) {
-            bus.emit('compiler-warning', { json, compiler: comp });
+          const compilerErrorEventName = 'compiler-error';
```

Comment:
this is very microsofty :)

would prefer:

```
const events = {
  error: 'compiler-error',
  warning: 'compiler-warning',
};
```

Reply:
Done